src/scrabble.js

```diff
@@ -1,5 +1,67 @@
-function scrabble() {
+const wordValues = {
+    _: 0,
+    A: 1, E: 1, I: 1, O: 1, U: 1, L: 1, N: 1, R: 1, S: 1, T: 1,
+    D: 2, G: 2,
+    B: 3, C: 3, M: 3, P: 3,
+    F: 4, H: 4, V: 4, W: 4, Y: 4,
+    K: 5,
+    J: 8, X: 8,
+    Q: 10, Z: 10
+}
+
+let score = 0
+
+function scrabble(word) {
+    const wordArray = sanitizeAndConvert(word)
+    for (let letter of wordArray) { score += wordValues[letter] }
+    return outputFinalResultAndReset(score)
+}
+
+function outputFinalResultAndReset(finalScore) {
+    score = 0
+    return finalScore
+}
+
+function sanitizeAndConvert(word) {
+    if (word === null) { return ['_'] }
+    const wordCheckedForMultples = checkForMultiples(word);
+    const sanitizedWord = wordCheckedForMultples.toUpperCase().replace(' ', '_').replace('\t', '_').replace('\n', '_')
+    if (sanitizedWord.length === 1) { return [`${sanitizedWord}`] }
+    return sanitizedWord.split('')
+}
```

Comment:
```js
function sanitizeAndConvert(word) {
    if (word === null) {
        return ['_']
    }
    
    const wordCheckedForMultples = checkForMultiples(word);
    
    const sanitizedWord = wordCheckedForMultples.toUpperCase()
        .replace(' ', '_')
        .replace('\t', '_')
        .replace('\n', '_')
        
    if (sanitizedWord.length === 1) {
        return [`${sanitizedWord}`]
    }
    
    return sanitizedWord.split('')
}
```

Write code as if the person who's going to maintain it is homicidal and knows where you live :)

Reply:
Added white space (though not quite as much) and updated PR.

Reply:
Added new solution based on today's morning session: new_scrabble.js (tests new_scrabble-spec.js).